INSTALL.md

```diff
@@ -0,0 +1,213 @@
+Environment="PATH=/home/artefactual/AIPscan/venv/bin"
+ExecStart=/home/artefactual/AIPscan/venv/bin/gunicorn --workers 3 --bind unix:aipscan.sock -m 007 wsgi:app
+
+[Install]
+WantedBy=multi-user.target
+```
+
+* Start the new AIPscan Gunicorn service:
+
+```bash
+sudo systemctl start aipscan
+sudo systemctl enable aipscan
+```
+
+* Check that it is running correctly:
+
+```bash
+sudo systemctl status aipscan
+```
+* You should see output that looks like:
+
+```bash
+● aipscan.service - Gunicorn instance to serve AIPscan
+   Loaded: loaded (/etc/systemd/system/aipscan.service; enabled; vendor preset: enabled)
+   Active: active (running) since Wed 2020-11-04 21:18:30 UTC; 3h 4min ago
+ Main PID: 25278 (gunicorn)
+    Tasks: 4 (limit: 4915)
+   CGroup: /system.slice/aipscan.service
+           ├─25278 /home/artefactual/AIPscan/venv/bin/python3 /home/artefactual/AIPscan/venv/bin/gunicorn --workers 3 --bind unix:aipscan.sock -m 007 wsgi:app
+           ├─25301 /home/artefactual/AIPscan/venv/bin/python3 /home/artefactual/AIPscan/venv/bin/gunicorn --workers 3 --bind unix:aipscan.sock -m 007 wsgi:app
+           ├─26969 /home/artefactual/AIPscan/venv/bin/python3 /home/artefactual/AIPscan/venv/bin/gunicorn --workers 3 --bind unix:aipscan.sock -m 007 wsgi:app
+           └─26985 /home/artefactual/AIPscan/venv/bin/python3 /home/artefactual/AIPscan/venv/bin/gunicorn --workers 3 --bind unix:aipscan.sock -m 007 wsgi:app
+```
+
+### Nginx web server
+
+* Install Nginx.
+
+```bash
+sudo apt update
+sudo apt install nginx
+```
+
+* Configure a Nginx server block for the AIPscan application. Again, replace
+the `/home/artefactual/AIPscan/aipscan.sock` path with the path to your
+AIPscan root directory (e.g. `/home/ubuntu/AIPscan/aipscan.sock`).
+
+```bash
+sudo nano /etc/nginx/sites-available/aipscan
+```
+
+* Add the following content to this file and save.
+
+```bashhttps://github.com/artefactual-labs/AIPscan/issues/72
+server {
+    listen 80;
+    server_name your.aipscan.server.ip.here;
+
+    location / {
+        satisfy all;
+
+        include proxy_params;
+        proxy_pass http://unix:/home/artefactual/AIPscan/aipscan.sock;
+    }
+}
+```
+
+* Create the symlink to enable this Nginx configuration:
+
+```bash
+sudo ln -s /etc/nginx/sites-available/aipscan /etc/nginx/sites-enabled
+```
+
+* Confirm that the Nginx configuration is correct:
+
+```bash
+sudo nginx -t
+```
+
+* You should see:
+
+```bash
+nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
+nginx: configuration file /etc/nginx/nginx.conf test is successful
+```
+
+* Don't forget to [enable your firewall][ufw-1] to protect the application:
+
+```bash
+sudo ufw status
+```
+
+* Also, be sure to restart Nginx anytime you make any changes to the server
+block:
+
+```bash
+sudo systemctl restart nginx
+```
+
+### Celery service
+
+* AIPscan uses Celery workers, coordinated by RabbitMQ to run background jobs.
+To run Celery as a persistent Ubuntu service, create the following file:
```

Comment:
I might suggest dropping "Ubuntu" so this just says "as a peristent service", since we specify OS support above and systemd can be used on a number of Linux distros.